relayer/src/chain.rs

```diff
@@ -361,6 +362,12 @@ pub trait Chain: Sized {
         request: QueryConnectionChannelsRequest,
     ) -> Result<Vec<ChannelId>, Error>;
 
+    /// Performs a query to retrieve the identifiers of all clients associated with a chain.
+    fn query_chain_clients(
```

Comment:
we are already in a chain context  so maybe a better name is:
```suggestion
    fn query_clients(
```